app/services/user_organisation_decliner.rb

```diff
@@ -0,0 +1,34 @@
+class UserOrganisationDecliner
+
+  def initialize(listener, user, current_user)
+    @listener = listener
+    @user = user
+    @current_user = current_user
+  end
+
+  def call
+    error_message_if_not_superadmin
+    remove_pending_org_from_user
+  end
+
+  attr_reader :listener, :user, :current_user
+
+  private
+
+  def is_current_user_superadmin?
```

Comment:
is this method really necessary - feels like it's just repeating the already nicely descrptive `current_user.superadmin?`